### PR TITLE
Could not write to log file

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

**Comment by garvinhicking** [comment]
Deleting log files may cause a loss of important data. I would rather like to understand the cause of this, not the symptom.

I suppose that either your var/log/ directory or the file inside there was missing write permissions? Then the suggestion would need to be to adjust write permissions for the TYPO3 webserver user/process.

Or was your disk full? In that case the recommendaton would be to ensure enough diskspace, or to utilize logfile rotation.

**Comment by linawolf** [comment]
@garvinhicking good point, added some more steps that could be checked

File: Documentation/Exceptions/1345036335.rst [hunk 1]
```diff
@@ -7,18 +7,20 @@ TYPO3 Exception 1345036335
...
-   --------
+Solution
+--------
 
-   1. Did you find a solution?
-   2. Explain what you did ..
+#.  If the Install Tool can be opened, go to :guilabel:`Admin Tools >  Environment > Directory Status`
+    and check the directory status.
+#.  Check if `typo3temp/var/log` and all files therein are writable for TYPO3
+#   If the disk is full, your might want to switch to log rotation.
```
Comment (garvinhicking) [inline Documentation/Exceptions/1345036335.rst:25]:
```suggestion
#   If the disk is full, you might want to switch to log rotation.
```

**Comment by franzholz** [comment]
I could repeat now this issue. So I attach the debug output here.
It happens when I have kept a TYPO3 backend session open for a very long time. Now the relogin box appears. After I type my password and press the button, I come into this exception.

[Exception-1345036335.html.txt](https://github.com/user-attachments/files/18267185/Exception-1345036335.html.txt)

**Comment by franzholz** [comment]
Here is the answer:

```
user:/homepath/typo3temp/var/log$ ls -ltr
total 4384
-rw-rw-r-- 1 dcp307300028 dcp307300028    1142 Jun 26  2024 tcpdfbill_tt_products.log
-rw-rw-r-- 1 dcp307300028 dcp307300028 2854912 Dec 28 20:31 typo3_deprecations_4b4937a2ac.log
-rw-rw-r-- 1 dcp307300028 dcp307300028 1622016 Dec 28 20:31 typo3_4b4937a2ac.log
user:/homepath/typo3temp/var/log$ 

```

So the log file still can be written despite of this exception. 

If I repeat a click in the backend, then the time of the log files gets to the current time. However the file size does not increase.

**Comment by franzholz** [comment]
Now I have done this:

`chmod -R 777 typo3temp
`

Hm, it still shows the exception which blocks the TYPO3 backend.

**Comment by franzholz** [comment]
Now I have checked the server tool.

_Verfügbarer Speicherplatz:0 MB von 12,00 GB_

**Pull request merged**